packages/clamav-rest-server/jest-mongodb-config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Is this file used?   The clamav-rest-server doesn't depend on mongo right?

Reply:
Deleted